nistats/reporting.py

```diff
@@ -6,15 +6,170 @@
 
 import os
 import warnings
+from string import ascii_lowercase
+
 import numpy as np
+import pandas as pd
 from scipy import stats
 import nilearn.plotting  # overrides the backend on headless servers
+from nilearn.image.resampling import coord_transform
 import matplotlib
 import matplotlib.pyplot as plt
+from scipy import ndimage
 from patsy import DesignInfo
+
 from .design_matrix import check_design_matrix
 
 
+def _local_max(data, min_distance):
+    """Find all local maxima of the array, separated by at least min_distance.
+    Adapted from https://stackoverflow.com/a/22631583/2589328
+
+    Parameters
+    ----------
+    data : array_like
+        3D array of with masked values for cluster.
+
+    min_distance : :obj:`int`
```

Comment:
I'm switching to mm and including the affine. I don't think it's efficient to use an image (because it would mean generating one for each cluster in get_clusters_table), but it might increase readability. Do you have a preference?

Reply:
Passing the affine is fine imho.